src/scrabble.js

```diff
@@ -1,5 +1,46 @@
-function scrabble() {
+const { empty } = require("joi/lib/types/alternatives")
+let letters = {
```

Comment:
can use const as letters won't change

Reply:
+1 to Sam's feedback here. Also, letters could be defined outside of scrabble() to make it a global variable, so if you start breaking down scrabble into smaller functions with the extensions in the future, those other functions will also have access to it.